shared/util/local_processor.go

```diff
@@ -34,11 +35,19 @@ func (s *LocalProcessor) Tarzip(src string, f *os.File) error {
 	defer tw.Close()
 
 	return filepath.Walk(src, func(file string, fi os.FileInfo, err error) error {
+		dir := strings.Split(filepath.Dir(file), "/")[0]
+		logrus.Debugf("Processing file %s (%s) ...", fi.Name(), file)
 		if err != nil {
 			return errors.Wrapf(err, "Unable to walk the file path %s", file)
 		}
 
 		if !fi.Mode().IsRegular() {
+			logrus.Debugf("Skipping file (%s) ...", fi.Name())
+			return nil
+		}
+
+		if fi.Name() == ".DS_Store" || fi.Name() == ".terraform" || fi.Name() == ".git" || fi.Name() == ".terraform.lock.hcl" || filepath.Ext(fi.Name()) == ".tar.gz" || dir == ".terraform" || dir == ".git" {
```

Comment:
can you add these into a slice and loop over them rather than a large conditional like this? something like

```suggestion
		ignored:=[]string{".DS_Store", "blah", "blah"}
		for _, i := range ignored {
		  return nil
		}
```